examples/topics/types/csv/csv_to_parquet_100k.php

```diff
@@ -11,6 +11,12 @@
 
 require __DIR__ . '/../../../bootstrap.php';
 
+if (!\file_exists(__FLOW_OUTPUT__ . '/dataset.csv')) {
+    print "Data file is missing, please first run: php examples/setup/php_to_csv_and_json.php\n";
```

Comment:
in that case, can we just run the other example called `php_to_csv_and_jsonphp` ?

Reply:
Done.